backends/ebpf/tests/ptf/test.py

```diff
@@ -389,3 +389,112 @@ def runTest(self):
         self.counter_verify(name="ingress_test2_cnt", keys=[0x1fe], packets=1)
         self.counter_verify(name="ingress_test3_cnt", keys=[0x1fe], bytes=199, packets=1)
         self.counter_verify(name="ingress_action_cnt", keys=[5], bytes=299, packets=2)
+
+
+class RegisterActionPSATest(P4EbpfTest):
```

Comment:
These test were in a separate file called `register.py`, maybe preserve this?

Reply:
I'd like to avoid this. Keeping all PTF tests in one file is good enough IMO.